examples/tutorial_examples/add_custom_bundle.py

```diff
@@ -0,0 +1,233 @@
+            'https://github.com/yeatmanlab/AFQ/raw/c762ca4c393f2105d4f444c44d9e4b4702f0a646/SLF123/ROIs/PaR.nii.gz',
+            'https://github.com/yeatmanlab/AFQ/raw/c762ca4c393f2105d4f444c44d9e4b4702f0a646/SLF123/ROIs/PrgL.nii.gz',
+            'https://github.com/yeatmanlab/AFQ/raw/c762ca4c393f2105d4f444c44d9e4b4702f0a646/SLF123/ROIs/PrgR.nii.gz',
+            'https://github.com/yeatmanlab/AFQ/raw/c762ca4c393f2105d4f444c44d9e4b4702f0a646/SLF123/ROIs/SFgL.nii.gz',
+            'https://github.com/yeatmanlab/AFQ/raw/c762ca4c393f2105d4f444c44d9e4b4702f0a646/SLF123/ROIs/SFgR.nii.gz',
+            'https://github.com/yeatmanlab/AFQ/raw/c762ca4c393f2105d4f444c44d9e4b4702f0a646/SLF123/ROIs/SLFt_roi2_L.nii.gz',
+            'https://github.com/yeatmanlab/AFQ/raw/c762ca4c393f2105d4f444c44d9e4b4702f0a646/SLF123/ROIs/SLFt_roi2_R.nii.gz']
+
+
+#############################################################################
+# Get ROIs and save to disk
+# --------------------------------
+# The goal of this tutorial is to demostrate how to segment new pathways based
+# on ROIs that are saved to disk. We'll start off by donwloading some ROIs that are saved online
+
+# Define and create the directory for the template ROIs
+# op.expanduser("~") expands your ~ directory into the full string
+# and op.join joins these paths, to make ~/AFQ_data/SLF_ROIs/
+template_dir = op.join(
+    op.expanduser("~"),
+    'AFQ_data/SLF_ROIs/')
+os.makedirs(template_dir, exist_ok=True)
+
+# Download the ROI files
+for roi_url in roi_urls:
+    wget.download(roi_url, template_dir)
+
+
+#############################################################################
+# Define custom `BundleDict` object
+# --------------------------------
+# The `BundleDict` object holds information about "include" and "exclude" ROIs,
+# as well as endpoint ROIS, and whether the bundle crosses the midline. In this
+# case, the ROIs are all defined in the MNI template space that is used as the
+# default template space in pyAFQ, but, in principle, other template spaces
+# could be used. In this example, we provide paths to the ROIs to populate the BundleDict
+
+bundles = abd.BundleDict({
+    "L_SLF1": {
+        "include": [
+            template_dir + 'SFgL.nii.gz',
+            template_dir + 'PaL.nii.gz'],
+        "exclude": [
+            template_dir + 'SLFt_roi2_L.nii.gz'],
+
+        "cross_midline": False,
+    },
+    "L_SLF2": {
+        "include": [
+            template_dir + 'MFgL.nii.gz',
+            template_dir + 'PaL.nii.gz'],
+        "exclude": [
+            template_dir + 'SLFt_roi2_L.nii.gz'],
+
+        "cross_midline": False,
+    },
+    "L_SLF3": {
+        "include": [
+            template_dir + 'PrgL.nii.gz',
+            template_dir + 'PaL.nii.gz'],
+        "exclude": [
+            template_dir + 'SLFt_roi2_L.nii.gz'],
+
+        "cross_midline": False,
+    }
+
+})
```

Comment:
Just removing some white-space in-between lines:
```suggestion
bundles = abd.BundleDict({
    "L_SLF1": {
        "include": [
            template_dir + 'SFgL.nii.gz',
            template_dir + 'PaL.nii.gz'],
        "exclude": [
            template_dir + 'SLFt_roi2_L.nii.gz'],
        "cross_midline": False,
    },
    "L_SLF2": {
        "include": [
            template_dir + 'MFgL.nii.gz',
            template_dir + 'PaL.nii.gz'],
        "exclude": [
            template_dir + 'SLFt_roi2_L.nii.gz'],
        "cross_midline": False,
    },
    "L_SLF3": {
        "include": [
            template_dir + 'PrgL.nii.gz',
            template_dir + 'PaL.nii.gz'],
        "exclude": [
            template_dir + 'SLFt_roi2_L.nii.gz'],
        "cross_midline": False,
    }
})
```